src/popover/popover.directive.ts

```diff
@@ -33,6 +41,10 @@ export class PopoverDirective implements OnInit, OnDestroy {
    * Currently only supports "body".
    */
   @Input() public container: string;
+  /**
+   * Specifies if popover should be closed after clicking outside
+   */
+  @Input() public popoverCloseOnClickOutside: boolean;
```

Comment:
may be better to make it a part of config?
to not blow html?
and remove @input